datajunction-server/datajunction_server/api/authentication/basic.py

```diff
@@ -9,7 +9,7 @@
 from fastapi.security import OAuth2PasswordRequestForm
 from sqlmodel import Session, select
 
-from datajunction_server.constants import DJ_AUTH_COOKIE, DJ_LOGGED_IN_FLAG_COOKIE
+from datajunction_server.constants import AUTH_COOKIE, LOGGED_IN_FLAG_COOKIE
```

Comment:
Is your thinking here that these env variables are generic and not DJ specific?

Reply:
Yeah the constant names were pretty long and it seemed a bit silly to have the DJ_ prefix since there aren't any other cookies that would make this ambiguous.